tests/integration/pilot/outboundtrafficpolicy/helper.go

```diff
@@ -166,3 +173,38 @@ func RunExternalRequestTest(expected map[string][]string, t *testing.T) {
 			}
 		})
 }
+
+func clusterName(target echo.Instance, port echo.Port) string {
```

Comment:
Doesn't have to be done here, but wee should probably lift this up to somewhere common.